aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -299,6 +309,18 @@ class WriterJsonTar(ArchiveWriterAbstract):
     The entire containing folder is then compressed as a tar file.
     """
 
+    def __init__(self, filename: str, progress_context: Optional[ProgressContext] = None, **kwargs: Any):
+        """A writer for zipped archives.
+
+        :param filename: the filename (possibly including the absolute path)
+            of the file on which to export.
+        :param progress_context: A context manager for creating and updating a progress bar
```

Comment:
also here expanding the description would make semse

Reply:
changed